src/main/java/com/redhat/management/approval/Approver.java

```diff
@@ -1,64 +1,66 @@
 package com.redhat.management.approval;
 
+import java.io.Serializable;
+
 /**
  * This class was automatically generated by the data modeler tool.
  */
 
-public class Approver implements java.io.Serializable {
+public class Approver implements Serializable {
 
     static final long serialVersionUID = 1L;
 
-    private java.lang.Integer id;
-    private java.lang.String firstName;
-    private java.lang.String lastName;
-    private java.lang.String userName;
-    private java.lang.String emailAddress;
+    private int id;
+    private String firstName;
+    private String lastName;
+    private String userName;
+    private String emailAddress;
 
     public Approver() {
     }
 
-    public java.lang.Integer getId() {
+    public int getId() {
         return this.id;
     }
 
-    public void setId(java.lang.Integer id) {
+    public void setId(int id) {
         this.id = id;
     }
 
-    public java.lang.String getFirstName() {
+    public String getFirstName() {
         return this.firstName;
     }
 
-    public void setFirstName(java.lang.String firstName) {
+    public void setFirstName(String firstName) {
         this.firstName = firstName;
     }
 
-    public java.lang.String getLastName() {
+    public String getLastName() {
         return this.lastName;
     }
 
-    public void setLastName(java.lang.String lastName) {
+    public void setLastName(String lastName) {
         this.lastName = lastName;
     }
 
-    public java.lang.String getUserName() {
+    public String getUserName() {
         return this.userName;
     }
 
-    public void setUserName(java.lang.String userName) {
+    public void setUserName(String userName) {
         this.userName = userName;
     }
 
-    public java.lang.String getEmailAddress() {
+    public String getEmailAddress() {
         return this.emailAddress;
     }
 
-    public void setEmailAddress(java.lang.String emailAddress) {
+    public void setEmailAddress(String emailAddress) {
         this.emailAddress = emailAddress;
     }
 
-    public Approver(java.lang.String userName, java.lang.String emailAddress, 
-            java.lang.String firstName, java.lang.String lastName) {
+    public Approver(String userName, String emailAddress, 
```

Comment:
I prefer to have all the arguments in one line. What do you like?

In new line 71, remove the leading `\n`, i.e., return `"Approver:" +`